migrations/2023-02-07-070512_change_merchant_connector_id_data_type/up.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TABLE merchant_connector_account
+ALTER COLUMN merchant_connector_id TYPE VARCHAR(128) USING merchant_connector_id::varchar;
+
+
+ALTER TABLE merchant_connector_account
+ALTER COLUMN merchant_connector_id DROP DEFAULT;
```

Comment:
So since in down.sql it creates default to sequence this is needed

Reply:
Would this `DEFAULT` constraint be present when `diesel migration run` is run on an empty database as well?

Reply:
Yes it will be there.